app/assets/javascripts/admin/controllers/admin-users-list-show.js.es6

```diff
@@ -88,7 +88,12 @@ export default Ember.Controller.extend(CanCheckEmails, {
     },
 
     showEmails: function() {
-      this.set("showEmails", true);
+      this.set("showEmailsToggle", true);
```

Comment:
are we able to utilize `this.toggleProperty` here instead of having two actions?